cmds/db-manager/main.go

```diff
@@ -239,8 +246,6 @@ func (m *MyMigrate) MigrationDirection(desiredVersion string, desiredStep int) (
 	if err != nil {
 		return 0, fmt.Errorf("Failed to get current DB version to determine migration direction: %v", err)
 	}
-	if !semver.IsValid(currentVersion) {
-		return 0, fmt.Errorf("The current DB Version format is in valid")
-	}
-	return Direction(semver.Compare(desiredVersion, currentVersion)), nil
+
+	return Direction(desiredVersion.Compare(*currentVersion)), nil
```

Comment:
Do you have to cover the case where desiredVersion == nil?

Reply:
Nope, `desiredVersion` is not a pointer.